src/uniprotkb/components/entry/GoCam.tsx

```diff
@@ -135,12 +135,28 @@ const GoCam = ({ primaryAccession }: Props) => {
       abortController.abort();
     };
   }, [goCamIdToItem, primaryAccession, setGoCamIdToNode]);
+
   useEffect(() => {
     if (uniprotGoCamIds?.[0]) {
       setSelectedId(uniprotGoCamIds[0]);
     }
   }, [uniprotGoCamIds]);
 
+  const showGoCamViz =
```

Comment:
A quick visual check made me find out that there's an issue where if the screen is small and deactivates this boolean we will see the "UniProt curated GO-CAM models have not been found for this entry within the Gene Ontology knowledgebase." sentence. I will stop reviewing this for today, just letting you know if you want to have a look tomorrow morning.
Not sure what should happen, we have multiple options:
- Display useful fallback without the visualisation (just a link to GO-Cam website with the models?)
- Don't display the tab at all
- Display a click-to-load button (see structure section logic below in the page)
- Something else?